tests/unit/kvdb/omf_version_test.c

```diff
@@ -32,7 +32,7 @@ MTF_DEFINE_UTEST(omf_version_test, global_version)
     ASSERT_EQ(MBLOCK_METAHDR_VERSION, 2);
     ASSERT_EQ(MDC_LOGHDR_VERSION, 2);
     ASSERT_EQ(WAL_VERSION, 1);
-    ASSERT_EQ(KVDB_META_VERSION, 1);
+    ASSERT_EQ(KVDB_META_VERSION, 2);
```

Comment:
Suspicious to bump this version w/o bumping the global OMF version.
Do you have migration tests?

Reply:
Ahh... the kvdb meta version is the one omf version that is not "under" the global version?
If so, then perhaps it makes sense to test the KVDB_META_VERSION in the global version test.

Reply:
I had bumped the global OMF version in my last update to the mpool OMF.. This PR updates the last OMF component for the pmem work. I didn't want to bump the global OMF version again just because I'm submitting the changes in two separate PRs but both targeting a single logical task.

Reply:
I did migration tests manually, but don't have any automated tests yet.

Reply:
AFAIR our discussions, the kvdb meta omf version also contributes to the global omf version..

Reply:
Okay.  That makes sense.  It really only needs to be bumped once per release.